### PR TITLE
Archive 2021 Release notes + Flutter 2.0.1 release notes

### PR DESCRIPTION
**Pull request by PravinPK**
Please do not merge yet.. 
Once more update to Assurance Flutter extension is on the way... need to update the release notes

### CODE REVIEW

File: release-notes/2021.md [hunk 1]
```diff
@@ -0,0 +1,758 @@
+---
+description: Archived Release Notes for 2021
+---
+
+# 2020
```
Comment (praveek) [inline release-notes/2021.md:5]: 2021

File: release-notes/README.md [hunk 2]
```diff
@@ -4,755 +4,19 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
-* Ability to create XDM Experience Events and send them to Experience Edge Network. An optional Experience Edge response callback can be registered per event.
-* The XDM Experience Events can be sent to individual Adobe Experience Platform datasets when a custom dataset identifier is set at event level.
-* ECID is automatically attached on each XDM Experience Event request.
-* Integration with Assurance extension enabling new insights in Project Griffon about the XDM Experience Event processing and XDM data validation for an improved validation and debugging experience.
-* Detailed warning/error messages are available through Project Griffon UI and logs.
-
-This SDK is compatible with Android Core 1.5.7 and above.
-
-### Adobe Experience Platform Edge Network Launch Extension 1.0.2
+Upgrade the following flutter package to reference Android libraries from mavenCentral :
```
Comment (emdobrin) [inline release-notes/README.md:11]: Updated the following flutter packages to reference the Android libraries from Maven Central Repository:

File: release-notes/README.md [hunk 3]
```diff
@@ -4,755 +4,19 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+* [Flutter ACPCore v2.0.1](https://pub.dev/packages/flutter_acpcore/versions/2.0.1)
+* [Flutter ACPUserProfile v2.0.1](https://pub.dev/packages/flutter_acpuserprofile/versions/2.0.1)
+* [Flutter ACPPlaces v2.0.1](https://pub.dev/packages/flutter_acpplaces/versions/2.0.1)
+* [Flutter ACPPlacesMonitor v2.0.1](https://pub.dev/packages/flutter_acpplaces_monitor/versions/2.0.1)
 
-## January 20, 2021
+### Flutter Assurance 2.0.1
 
-### iOS Audience 2.3.0
+- Migrate Android build from jcenter to mavenCentral
```
Comment (emdobrin) [inline release-notes/README.md:20]: Should we use a similar wording here:
Updated ACPAssurance package to reference the Android library from Maven Central repository.

File: release-notes/README.md [hunk 4]
```diff
@@ -4,755 +4,19 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+- Migrate to the new Android APIs based on [FlutterPlugin](https://api.flutter.dev/javadoc/io/flutter/embedding/engine/plugins/FlutterPlugin.html).
```
Comment (emdobrin) [inline release-notes/README.md:21]: Migrate -> Migrated
Do we need to include any additional info here on what does this change fixes?

Comment (PravinPK) [reply release-notes/README.md:21]: I believe this description should be fine.. reading from the flutter world.

PS : This change was made to accommodate for the deprecation issue : https://github.com/adobe/flutter_assurance/issues/21

**Review by emdobrin — APPROVED** [review]

**Pull request merged**